packages/components/Organisms/ImageGallery/ComplexImage.vue

```diff
@@ -0,0 +1,242 @@
+<template>
+  <div class="rpl-complex-image">
+    <h3 class="rpl-complex-image__title" v-if="title">{{ title }}</h3>
+    <rpl-figure v-if="image" :image="image" :caption="caption" :source="source" />
+    <div class="rpl-complex-image__controls">
+      <!-- Fullscreen modal button -->
+      <button class="rpl-complex-image__button" v-if="fullscreenLabel && image" @click="showModal = true">
+        <rpl-text-icon symbol="view" color="primary" :text="fullscreenLabel" placement="before" />
+      </button>
+      <!-- Markup expander -->
+      <button class="rpl-complex-image__button" v-if="expandLabel && !markupVisible" @click="markupVisible = true">
+        <rpl-text-icon symbol="table" color="primary" :text="expandLabel" placement="before" />
+      </button>
+      <div class="rpl-complex-image__expander" :class="{ 'rpl-complex-image__expander--open': markupVisible }">
+        <div class="rpl-complex-image__expander-header">
+          <rpl-text-icon class="rpl-complex-image__expander-title" symbol="table" color="primary" :text="expandTitleLabel" placement="before" />
+          <button class="rpl-complex-image__expander-close-button" @click="markupVisible = false">
+            <rpl-icon symbol="close" color="primary" size="s" />
+            <span class="rpl-complex-image__expander-close-button-text">{{ expandcloseText }}</span>
+          </button>
+        </div>
+        <div class="rpl-complex-image__expander-markup">
+          <rpl-markup :html="html" />
+        </div>
+      </div>
+      <!-- Download button -->
+      <a class="rpl-complex-image__button" v-if="downloadLabel && downloadFilename" :download="downloadFilename" :href="image.src">
+        <rpl-text-icon symbol="download" color="primary" :text="downloadLabel" placement="before" />
+      </a>
+    </div>
+    <!-- Fullscreen modal -->
+    <rpl-image-gallery-modal v-if="showModal" @close="showModal = false">
+      <template slot="body">
+        <rpl-fullscreen-image :image="image" :title="title" :caption="caption" />
+      </template>
+    </rpl-image-gallery-modal>
+  </div>
+</template>
+
+<script>
+import { RplIcon, RplTextIcon } from '@dpc-sdp/ripple-icon'
+import RplFigure from '@dpc-sdp/ripple-figure'
+import RplMarkup from '@dpc-sdp/ripple-markup'
+import { RplImageGalleryModal, RplFullscreenImage } from '@dpc-sdp/ripple-image-gallery'
+
+export default {
+  name: 'RplPublicationImage',
```

Comment:
@dylankelly isn't it suppose to be `RplComplexImage`?

Reply:
ah thanking you